dotnet-setversion/Program.cs

```diff
@@ -2,64 +2,62 @@
 using System.IO;
 using System.Linq;
 using System.Xml.Linq;
-using McMaster.Extensions.CommandLineUtils;
 
 namespace dotnet_setversion
 {
     class Program
     {
         static int Main(string[] args)
         {
-            var app = CreateApplication();
-            return app.Execute(args);
-        }
+            string csprojFile;
+            string versionString;
 
-        private static CommandLineApplication CreateApplication()
-        {
-            var app = new CommandLineApplication(throwOnUnexpectedArg: true)
+            bool IsHelpFlag(string arg) => arg == "-h" || arg == "--help";
+            bool IsVstsFlag(string arg) => arg == "--vsts";
```

Comment:
I don't quite understand this `--vsts` flag. Why does it take an additional flag to specify a .csproj file?
Wouldn't it be sufficient to assume "VSTS mode" when two arguments instead of one have been passed to the program?

Reply:
Haven't touched this in a while so a lot of is hazy, but I found the rationale for this in [this comment](https://github.com/TAGC/dotnet-setversion/issues/5#issuecomment-358127283).

Reply:
@roryprimrose Is this still relevant for you?

Reply:
That would be how I'd have implemented it too. However the motivation for this feature was to allow the tool to work in VSTS, which I've never used but apparently could throw spanners in the works if done like that.

If Rory doesn't respond or it's no longer relevant to him, I'll just not worry about VSTS and go with the simple solution you propose.

Reply:
We use TFS which is pretty much VSTS but on-premise AFAIK. In a larger repo, I see why the usage of a glob like `**/*.csproj` would be desirable.

What do you think about a `-r/--recursive` switch? When toggled, `setversion` recursively searches the current directory for csproj files and applies the given version to all files it found. This would fit my requirement as well and still allows to add a parameter for specifying a single file later on if explicitly requested.

E.g.: `dotnet setversion -r 1.2.3` would result in output like this:
````
Version set to '1.2.3' in
C:\build\Project A.csproj
C:\build\Project B.csproj
C:\build\Module A\Module A.csproj
C:\build\Module B\Mobule B.csproj
...